src/components/banners/HomePageBannerForMobile.tsx

```diff
@@ -1,52 +1,57 @@
 import { GlobalDialog, useGlobalDialog } from '@/contexts/GlobalUIManager';
 import { getImage } from '@/lib/getImage';
-import { CheckCircle } from '@phosphor-icons/react';
+import { CheckCircle} from '@phosphor-icons/react';
 import Image from 'next/legacy/image';
+import Link from 'next/link';
 import { useRouter } from 'next/router';
 
-export const HomePageBannerForMobile = () => {
+export const HomePageBannerForMobile = (redirectTo) => {
```

Comment:
Remove this unused parameter. And if you want to pass a prop to a component, you must add the prop inside square brackets ({}).